roles/ceph-nfs/tasks/start_nfs.yml

```diff
@@ -9,7 +15,7 @@
     - ceph_nfs_rados_backend
 
 - name: create an empty rados index object
-  shell: "echo | rados -p {{ cephfs_data }} --cluster {{ cluster }} put {{ ceph_nfs_rados_export_index }} -"
+  shell: "{{ docker_exec_cmd_nfs | default('') }} rados -p {{ cephfs_data }} --cluster {{ cluster }} put {{ ceph_nfs_rados_export_index }} /dev/null"
```

Comment:
ditto

Reply:
the delegation is already present for this task.